src/app/zone-ingresses/views/item/ServicesView.vue

```diff
@@ -0,0 +1,134 @@
+<template>
+  <RouteView
+    v-slot="{ t }"
+    name="zone-ingress-services-view"
+  >
+    <AppView>
+      <template #title>
+        <h2>
+          <RouteTitle
+            :title="t('zone-ingresses.routes.item.navigation.zone-ingress-services-view')"
+            :render="true"
+          />
+        </h2>
+      </template>
+
+      <KCard class="mt-4">
+        <template #body>
+          <template
+            v-for="items in [
+              ...(props.data?.zoneIngress?.availableServices ? [props.data.zoneIngress.availableServices] : [])
```

Comment:
**Suggestion**: Please use a computed property here. They should be preferred over in-template operations like this. For one, they results are cached if their dependencies don’t change and avoid needless Vue rendering cycles. In-template logic breaks that. 

For another, this is just terrible to read.

Reply:
I swapped this out by guaranteeing `availableServices` is always set in the data layer before it ever gets here. Now I just access the property so no need for extra caching or checking for things